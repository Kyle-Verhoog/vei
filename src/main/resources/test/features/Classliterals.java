// FAIL
// Java 1.3
public class Classliterals {
  protected java.lang.Class x = A.class;
}