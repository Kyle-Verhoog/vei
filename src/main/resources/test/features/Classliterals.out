PUBLIC ┃ public
CLASS ┃ class
IDENTIFIER ┃ A
{ ┃ {
PROTECTED ┃ protected
IDENTIFIER ┃ java
. ┃ .
IDENTIFIER ┃ lang
. ┃ .
IDENTIFIER ┃ Class
IDENTIFIER ┃ x
= ┃ =
IDENTIFIER ┃ A
. ┃ .
CLASS ┃ class
; ┃ ;
} ┃ }